src/pint/models/timing_model.py

```diff
@@ -228,6 +221,62 @@ def __init__(self, name="", components=[]):
         self.add_param_from_top(
             MJDParameter(name="FINISH", description="End MJD for fitting"), ""
         )
+        self.add_param_from_top(
+            strParameter(
+                name="INFO",
+                description="Tells TEMPO to write some extra information about frontend/backend combinations; -f is recommended",
+            ),
+            "",
+        )
+        self.add_param_from_top(
+            strParameter(
+                name="TIMEEPH",
+                description="Time ephemeris to use for TDB conversion; for PINT, always FB90",
+            ),
+            "",
+        )
+        self.add_param_from_top(
+            strParameter(
+                name="T2CMETHOD",
+                description="Something to do with time conversion; for PINT, always TEMPO",
+            ),
+            "",
+        )
+        self.add_param_from_top(
+            boolParameter(
+                name="DILATEFREQ",
+                value=False,
+                description="Tells TEMPO2 o do something with frequencies; PINT always acts like it is N",
```

Comment:
"Whether or not to apply gravitational redshift and time dila-tion to observing frequency (Y/N)."

Reply:
In tempo2.

Reply:
Aha thanks!